client/layout/controller.js

```diff
@@ -43,24 +45,31 @@ const getPages = () => {
 		pages.push( {
 			container: Dashboard,
 			path: '/',
+			breadcrumbs: [ __( 'Dashboard', 'woocommerce-admin' ) ],
 			wpOpenMenu: 'toplevel_page_woocommerce',
 		} );
 	}
 
 	if ( window.wcAdminFeatures.analytics ) {
-		pages.push( {
-			container: Analytics,
-			path: '/analytics',
```

Comment:
This just renders and empty page. I've removed it since there's no link that points to it. Is it important?

Reply:
Clicking "Analytics" simply goes to the revenue page. Maybe this was a leftover from earlier times? Safe to remove.

![Screen Shot 2019-07-04 at 3 21 15 PM](https://user-images.githubusercontent.com/1922453/60637692-86b3ce80-9e6f-11e9-9a81-defe22c810d3.png)